skimage/color/colorconv.py

```diff
@@ -751,7 +751,8 @@ def rgbcie2rgb(rgbcie):
 
 
 def rgb2gray(rgb):
-    """Compute luminance of an RGB image.
+    """Compute luminance of an RGB image,
+    according to ITU-R BT.709-6 (2015) recommendation.
```

Comment:
Please move this line to a Notes section (see [numpy documentation guide](https://docs.scipy.org/doc/numpy/docs/howto_document.html)) and add a versionchanged tag for 0.15. Here's an [example](https://github.com/numpy/numpy/blob/v1.15.0/numpy/ma/extras.py#L1618). In the note, could you please note that prior to 0.15, the earlier coefficients were used? This will help future users to understand any discrepancy in their results between versions.

Reply:
Done. Thanks @jni !